File: API/Entity/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace API.Entity
{
    public class Product
    {
        public int Id { get; set; }        
        public string Skn { get; set; }        
        public string ProductName { get; set; }
        public string ProductDesc { get; set; }        
        public Category Category { get; set; }
        public int CategoryId { get; set; }
        public Brand Brand { get; set; }
        public int BrandId { get; set; }
        public ICollection<Photo> Photos { get; set; }
        public int RecommendedMinimumAge { get; set; }
        public string RecommendedGender { get; set; }
        public double ProductPrice { get; set; }
        public int AvailableQty { get; set; }
        public int MinOrderQty { get; set; }
        public DateTime ProductCreatedDate { get; set; }
        public DateTime LastUpdatedDate { get; set; }
    }
}

File: client/src/app/_models/product.ts
import { Brand } from "./brand";
import { Category } from "./category";
import { Photo } from "./Photo";

export interface Product {
    id: number;
    skn: string;
    productName: string;
    productDesc: string;
    photoUrl: string;
    category: Category;
    brand: Brand;
    photos: Photo[];
    recommendedMinimumAge: number;
    recommendedGender: string;
    productPrice: number;
    availableQty: number;
    minOrderQty: number;
  }

 
  
  
  